src/components/ChecTagsField.vue

```diff
@@ -237,6 +238,12 @@ export default {
       // input value is automatically added as a new tag.
       this.handleAddNewTag(e);
     },
+    handleDismissFocused() {
+      this.tagsFieldFocused = true;
+      if (this.tagsFieldFocused) {
```

Comment:
Won't this never be `false`?

Reply:
Yes true, i'll just take out the conditional